rkt/run.go

```diff
@@ -445,7 +445,9 @@ func (e *envFileMap) Set(s string) error {
 
 	scanner := bufio.NewScanner(file)
 	for scanner.Scan() {
-		// TODO parse '\' to skip new line character
+		if len(scanner.Text()) == 0 {
+			continue
```

Comment:
I agree, I don't want to implicitly set empty variables if the user does not write the `=`.

But IMO this is a detail, and it's not related to the issue being fixed.